infra/monitoring/main.tf

```diff
@@ -58,21 +58,25 @@ data "aws_iam_policy_document" "push_metric_policy" {
 }
 
 resource "aws_iam_policy" "push_metric_policy" {
-  name        = "push-metric-policy"
+  name        = var.module_name == "hudi" ? "push-metric-policy-hudi" : "push-metric-policy-json"
   path        = "/"
   description = "Allow push metrics to CloudWatch"
 
   policy = data.aws_iam_policy_document.push_metric_policy.json
 }
 
+
+
```

Comment:
Remove lines 68 and 69, there is already a line break (line 67). I think this would be done automatically with `terraform fmt`